utils/binpath.go

```diff
@@ -51,6 +56,18 @@ func GetBinaryPath(command string) (string, error) {
 		}
 	}
 
+	// look for the binaries in brew symlink directories
+	// example: /usr/local/opt/e2fsprogs/bin, /usr/local/opt/mtools/bin etc.
+	for _, p = range BrewSpecificPaths {
+		items, err := os.ReadDir(p)
+		for _, d := range items {
+			p, err = exec.LookPath(path.Join(d, "bin",  command))
```

Comment:
`e2fsprogs`, for example, have some binaries in `sbin` so we need to check that too.

Reply:
In the function 1st it will check in PATH, then in `ExternalBinaryPaths` which covers `/sbin`, then it will evaluate `/usr/ocal/opt`. So in case of `e2fsprogs` it should pick it up from `ExternalBinaryPaths` existing already.